javascript/javascript1/week2/week2-guzide/flight-booking/FligthBooking.js

```diff
@@ -0,0 +1,30 @@
+function getFullname(
+  firstname,
+  surname,
+  useFormalName = false,
+  gender = "female"
+) {
+  firstname = firstname || "";
```

Comment:
I think you do not need these two lines, if the intention was to test if it is an empty string then line 10 does that.